x-pack/plugins/security_solution/server/endpoint/routes/metadata/handlers.ts

```diff
@@ -72,11 +64,7 @@ export const getLogger = (endpointAppContext: EndpointAppContext): Logger => {
   return endpointAppContext.logFactory.get('metadata');
 };
 
-const errorHandler = <E extends Error>(
-  logger: Logger,
-  res: KibanaResponseFactory,
-  error: E
-): IKibanaResponse => {
+const errorHandler = <E extends Error>(res: KibanaResponseFactory, error: E): IKibanaResponse => {
```

Comment:
I was surprised to see that `logger` was not being used. That's probably a bug (maybe). the idea behind this general errorHandler is to be able to log errors as well as handle "known" errors by returning appropriate HTTP status codes.

Suggestion: Leave `logger` here and ensure that this handler logs the error that it is called with (probably should be inserted right at the start here).

Reply:
Line 78 below still have boom error checks. Should we remove it?

Reply:
done e58af8cdd7db95efa949586e4d130cdc5848e969